test_snappy.py

```diff
@@ -35,6 +35,15 @@
 from unittest import TestCase
 
 
+class SnappyModuleTest(TestCase):
+    def test_version(self):
+        assert tuple(map(int, snappy.__version__.split("."))) >= (0, 6, 1)
+        # Make sure that __version__ is identical to the version defined in setup.py
```

Comment:
It is good to test that it is the same, as it should be. Some projects use external tools like versioneer or poetry to keep these in sync. Here, it would be manual.